repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl

```diff
@@ -17,24 +18,16 @@
 <#-- Content Page Pixel -->
 <@contentPixel document.getCanonicalUUID() document.title></@contentPixel>
 
+<@headerBanner document />
 <article class="article article--filtered-list api-catalogue">
     <div class="grid-wrapper grid-wrapper--article">
-        <div class="grid-row">
-            <div class="column column--two-thirds column--reset">
-                <div class="article-header article-header--secondary">
-                    <h1 data-uipath="document.title">${document.title}</h1>
-                </div>
-            </div>
-        </div>
-        APi Catalogue - Preview
         <div class="grid-row">
             <div class="column column--two-thirds column--reset">
                 <#-- [FTL-BEGIN] 'Summary and optional Body' sections -->
                 <div id="section-summary" class="article-section article-section--summary  no-border">
                     <div class="grid-row">
                         <div class="column column--reset">
                             <div class="rich-text-content">
-                                <div data-uipath="website.linkslist.summary"><@hst.html hippohtml=document.summary contentRewriter=gaContentRewriter/></div>
```

Comment:
dont think i understand why this has been removed?

Reply:
This is now part of the headerBanner. its the same info